src/Rules/Install.hs

```diff
@@ -311,3 +312,22 @@ installIncludes = do
                   (destDir ++ ghcheaderDir ++ "/")
   where
     installHeader = installData -- they share same arguments
+
+installDocs :: Action ()
+installDocs = do
+    destDir <- getDestDir
+    docDir  <- installDocDir
+    root    <- buildRoot
+    installDirectory (destDir ++ docDir)
+
+    let usersGuide = root -/- "docs/pdfs/users_guide.pdf"
+    whenM (liftIO (IO.doesFileExist $ usersGuide)) $ installData [usersGuide] (destDir ++ docDir)
```

Comment:
Any particular reason to use non-Shake `doesFileExist`? We might as well track the existence `usersGuide`.

Reply:
Oh, I see. I will use `need` instead

Reply:
> I think you can just use Shake's doesFileExist and it will rerun this build rule if its result changes.

Right. This is still the right approach to go. I mean -- if `docs` related things are not built in the first place, then `install` shouldn't `need` them.